elasticai/creator/testing/ghdl_simulation.py

```diff
@@ -2,7 +2,7 @@
 import os
 import subprocess
 
-from .ghdl_report_parsing import parse_report
+from elasticai.creator.testing.ghdl_report_parsing import parse_report
```

Comment:
Stick to relative imports for intra-package imports please, ie.

```
from .ghdl_report_parsing import parse_report
```

That makes it very easy to move the full package and see at a glance that the import does not cross architectural boundaries.